app/views/user_mailer/admin_deleted_work_notification.html.erb

```diff
@@ -1,11 +1,11 @@
 <% content_for :message do %>
-  <p>Dear <%= @user.default_pseud.byline %>,</p>
+  <p><%= t '.hello', user: @user.default_pseud.byline %></p>
   
-  <p>Your work <i><%= style_bold(@work.title.html_safe) %></i> was deleted from the Archive by a site admin.</p>
+  <p><%= t('.html.part1', title: style_bold(@work.title.html_safe)).html_safe %></p>
```

Comment:
Yeah, that's what I meant. I'm looking at all the mailers right now, though, and I think we could actually do with a little refactoring so we don't have all these bold and italics tags scattered throughout our mailers. We have `style_bold` and `style_link` defined, but we should add something like:

```
  def style_bold_italics(text)
    ("<i><b style=\"color:#990000\">" + "#{text}".html_safe + "</b></i>").html_safe
  end

  # just guessing at the code here
  def style_link_bold_italics(body, url, html_options = {})
    html_options[:style] = "color:#990000"
    ("<i><b>" + link_to(body.html_safe, url, html_options) + "</i></b>").html_safe
  end
```

Not just because of translating, but because it makes it a lot easier to change in the future -- you only have to update the style in the helper, not change the individual mailer files. 

It doesn't need to happen in this pull request, of course. But it might be a good idea to do it before we finish the transition to translatable emails just because it's somewhat easier to _find_ all the places we do bold and italics now.